sdk/communication/azure-communication-email/README.md

```diff
@@ -75,15 +73,16 @@ message = {
     "recipients": {
         "to": [
             {
-                "email": "customer@domain.com",
+                "address": "customer@domain.com",
                 "displayName": "Customer Name"
             }
         ]
     },
-    "sender": "sender@contoso.com"
+    "senderAddress": "sender@contoso.com"
 }
 
-response = client.send(message)
+poller = email_client.begin_send(message)
+response = poller.result()
```

Comment:
We could maybe rename this, as it kinda looks like we're waiting on a "response" to the email we sent
```suggestion
result = poller.result()
```

Reply:
Good point, I'll change this.